### ISSUE TITLE
We can't add a new plugin - Error 404

### ISSUE THREAD

**Issue by cmoulliard** [fabric8io/fabric8-ipaas#366]
Example

<img width="725" alt="screenshot 2016-06-10 12 41 47" src="https://cloud.githubusercontent.com/assets/463790/15962052/c0125f04-2f08-11e6-9497-5b8724f7a2de.png">

This message appears after clicking on the `Add Plugin` button

<img width="1198" alt="screenshot 2016-06-10 12 43 18" src="https://cloud.githubusercontent.com/assets/463790/15962081/f79ae680-2f08-11e6-8ffc-9f7d097d97a7.png">

<img width="1469" alt="screenshot 2016-06-10 14 21 45" src="https://cloud.githubusercontent.com/assets/463790/15964042/b6859e2a-2f16-11e6-8685-1d7ac3dd04d6.png">

<img width="1529" alt="screenshot 2016-06-10 14 15 57" src="https://cloud.githubusercontent.com/assets/463790/15963946/fc153a96-2f15-11e6-965f-a78f282b2f39.png">

**Comment by cmoulliard** [comment]
https://gist.github.com/cmoulliard/c892617da07c01e18f058acbb2b9a5ae

**Comment by cmoulliard** [comment]
The resource can be downloaded 

````
stacktrace: "io.apiman.manager.api.rest.contract.exceptions.PluginNotFoundException: io.apiman.plugins:apiman-plugins-keycloak-oauth-policy:1.2.6.Final:war at io.apiman.manager.api.rest.impl.PluginResourceImpl.create(PluginResourceImpl.java:134) at io.apiman.manager.api.rest.impl.PluginResourceImpl$Proxy$_$$_WeldClientProxy.create(Unknown Source) at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) at java.lang.reflect.Method.invoke(Method.java:497) at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:139) at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:295) at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:249) at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:236) at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:395) at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:202) at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:221) at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:56) at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:51) at javax.servlet.http.HttpServlet.service(HttpServlet.java:847) at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:808) at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1669) at io.apiman.manager.api.micro.ManagerApiMicroServiceTxWatchdogFilter.doFilter(ManagerApiMicroServiceTxWatchdogFilter.java:57) at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) at io.apiman.common.servlet.RootResourceFilter.doFilter(RootResourceFilter.java:59) at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) at io.apiman.manager.api.security.impl.DefaultSecurityContextFilter.doFilter(DefaultSecurityContextFilter.java:56) at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) at io.fabric8.apiman.rest.Kubernetes2ApimanFilter.doFilter(Kubernetes2ApimanFilter.java:183) at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) at io.fabric8.apiman.rest.BearerTokenFilter.doFilter(BearerTokenFilter.java:113) at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) at io.fabric8.apiman.rest.BootstrapFilter.doFilter(BootstrapFilter.java:259) at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) at io.apiman.common.servlet.DisableCachingFilter.doFilter(DisableCachingFilter.java:59) at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) at io.apiman.common.servlet.ApimanCorsFilter.doFilter(ApimanCorsFilter.java:71) at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) at io.apiman.common.servlet.LocaleFilter.doFilter(LocaleFilter.java:61) at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585) at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:221) at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1127) at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515) at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185) at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1061) at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:110) at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97) at org.eclipse.jetty.server.Server.handle(Server.java:497) at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:310) at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257) at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540) at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635) at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555) at java.lang.Thread.run(Thread.java:745) Caused by: io.apiman.manager.api.core.exceptions.InvalidPluginException: Could not find plugin. (Not found locally and could not download from remote maven repositories) at io.apiman.manager.api.core.plugin.AbstractPluginRegistry.loadPlugin(AbstractPluginRegistry.java:107) at io.apiman.manager.api.micro.ManagerApiMicroServicePluginRegistry$Proxy$_$$_WeldClientProxy.loadPlugin(Unknown Source) at io.apiman.manager.api.rest.impl.PluginResourceImpl.create(PluginResourceImpl.java:130) ... 51 more "
````
even if we can have access to internet within the pod of apiman

```
oc rsh apiman-8pwtr
sh-4.2$ ls -la
total 12
drwxr-xr-x. 2 jboss jboss  59 Jul  1  2015 .
drwxr-xr-x. 4 root  root   55 Oct 21  2015 ..
-rw-r--r--. 1 jboss jboss  18 Mar  5  2015 .bash_logout
-rw-r--r--. 1 jboss jboss 193 Mar  5  2015 .bash_profile
-rw-r--r--. 1 jboss jboss 231 Mar  5  2015 .bashrc
sh-4.2$ ping repo1.maven.org
PING sonatype.map.fastly.net (23.235.43.209) 56(84) bytes of data.
64 bytes from 23.235.43.209: icmp_seq=1 ttl=61 time=40.0 ms
64 bytes from 23.235.43.209: icmp_seq=2 ttl=61 time=30.7 ms
^C
--- sonatype.map.fastly.net ping statistics ---
2 packets transmitted, 2 received, 0% packet loss, time 1000ms
rtt min/avg/max/mdev = 30.782/35.433/40.084/4.651 ms
<6.Final/apiman-plugins-keycloak-oauth-policy-1.2.6.Final.pom                
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3010  100  3010    0     0  13944      0 --:--:-- --:--:-- --:--:-- 14000
sh-4.2$ ls -la
total 20
drwxr-xr-x. 2 jboss jboss 4096 Jun 10 12:50 .
drwxr-xr-x. 4 root  root    55 Oct 21  2015 ..
-rw-r--r--. 1 jboss jboss   18 Mar  5  2015 .bash_logout
-rw-r--r--. 1 jboss jboss  193 Mar  5  2015 .bash_profile
-rw-r--r--. 1 jboss jboss  231 Mar  5  2015 .bashrc
-rw-r--r--. 1 jboss jboss 3010 Jun 10 12:50 apiman-plugins-keycloak-oauth-policy-1.2.6.Final.pom

```

**Comment by KurtStam** [comment]
Same fix as for #373
